src/phyltr/commands/assert.py

```diff
@@ -0,0 +1,18 @@
+from phyltr.commands.base import PhyltrCommand
+
+
+class Assert(PhyltrCommand):
```

Comment:
What's the motivation for introducing the `assert` command?  How does/should it differ from `stat`?

Reply:
https://github.com/lmaurits/phyltr/issues/19#issuecomment-463129307